msteams-platform/m365-apps/extend-m365-teams-personal-tab.md

```diff
@@ -233,6 +250,25 @@ Upon first run of local debug to Office or Outlook, you'll be prompted to sign i
 
 Provide feedback and report any issues with the Teams Toolkit debugging experience at [Microsoft Teams Framework (TeamsFx)](https://github.com/OfficeDev/TeamsFx/issues).
 
+#### Mobile debugging
+
+Here's how to remotely debug your app running in Office app for Android.
+
+   1. From your Android device, launch the Office app.
+   1. Open your profile **Me > Settings > Allow debugging**, and toggle on the option for **Enable remote debugging**.
+:::image type="content" source="images/office-android-enable-remote-debugging.png" alt-text="Screenshot showing Enable remote debugging":::
+
+        > [!NOTE]
+        > USB debugging is also an available option. Open the Developer Options screen on your Android. See [Configure On-Device Developer Options](https://developer.android.com/studio/debug/dev-options). Select Enable USB Debugging.
```

Comment:
Instead of a callout note, how about add this as the 1st step of the sequence:

1. If you're debugging on a physical Android device, connect it to your dev machine and enable the option for [USB debugging](https://developer.android.com/studio/debug/dev-options). (This is enabled by default with the Android emulator.)